test/distributed/fsdp/test_fsdp_param_exec_order_policy.py

```diff
@@ -271,12 +294,15 @@ def test_train_with_full_state_dict(
     @skip_if_lt_x_gpu(2)
     @parametrize("iters", [1, 3])
     def test_fsdp_flatten_params_exec_order(self, iters: int):
+        """
+        test the execution order of flatten parameters.
```

Comment:
Maybe we should make this a bit more specific. Is this testing the reverse gradient ready order path? If so, then maybe something like:
```suggestion
        Tests that ``FlatParameter`` s are reconstructed following reverse gradient
        ready order after the first iteration.
```
(rewrap accordingly)

Also, is this
https://github.com/awgu/pytorch/blob/2cdeefe0212fefe34ddcbe97b38e0afdcb66a531/test/distributed/fsdp/test_fsdp_param_exec_order_policy.py#L333
checking element-wise equality of the tensors in the list?

Reply:
yes